projects/ngx-feature-flag/src/lib/warning-page/feature-warning-page.component.less

```diff
@@ -1,43 +1,31 @@
 @import (reference) '../../assets/stylesheets/shared/main.less';
 
-.feature-level(@color-for-feature) {
-  &-page {
-    height: 100vh;
-    color: @color-pf-black-100;
-    background-color: @color-pf-black-800;
-    text-align: center;
-    .feature-content {
-      margin-top: 5%;
-      i {
-        color: @color-for-feature;
-      }
-      .btn-feature-level {
-        &:extend(.btn-default);
-        color: @color-pf-black-100;
-        border-color: @color-for-feature;
-        background-color: @color-for-feature;
-        background-image: linear-gradient(to bottom, @color-for-feature 0, @color-for-feature 100%);
-        &:hover {
-          background-color: @color-for-feature;
-          background-image: none;
-          border-color: darken((@color-for-feature), 10%);
-        }
-      }
-    }
-  }
+.warning-page-container {
+  margin-top: 40%;
+  display: inline-block;
+  padding-left: 40px;
+  padding-right: 40px;
```

Comment:
Why are you setting `display: inline-block` and the `padding`? Removing them in the demo app leads to a better looking card on smaller devices.